merlin_models/tf/core.py

```diff
@@ -2161,6 +2188,30 @@ def get_config(self):
         return config
 
 
+@tf.keras.utils.register_keras_serializable(package="merlin_models")
+class ModelBlock(Block, tf.keras.Model):
```

Comment:
Why is the `ModelBlock` needed? Just to make a `Block` serializable?

Reply:
Yeah it’s so that we can save each tower to disk.